message/infrastructure/sql/offsets_adapter.go

```diff
@@ -0,0 +1,64 @@
+package sql
+
+type OffsetsAdapter interface {
+	// AckQuery returns the SQL query that will mark a message as read for a given consumer group.
+	// Subscriber will not return those messages again for this consumer group.
+	AckMessageQuery(topic string, offset int, consumerGroup string) (string, []interface{})
```

Comment:
I changed approach to return query and args from one method, for me looks cleaner, what do you think?

Reply:
It's great, I really like how they've become more concise.
The only problem I have is that the return types `(string, []interface{})` are not immediately evindent from the method name or the godoc.

`XQueryAndArgs` or `XQueryArgs` doesn't seem like a great idea. So maybe in the godoc we could add something like `"AckMessageQuery" returns the SQL query and arguments that will mark a message as read...`

A little bit clearer and doesn't hinder readability that much

Reply:
Also, the godoc starts with a different name than the actual method name

Reply:
Also, the return types could be named `(query string, args []interface{})`.

Reply:
added info in doc about args, better now?

Reply:
:+1: